### ISSUE TITLE
Specifying more than one index that do not exist during a swap operation does not return the variant message of `index_not_found` returning the list of non-existent indexes.

### ISSUE THREAD

**Issue by gmourier** [meilisearch/meilisearch#2973]
**Describe the bug**

Specifying more than an index that does not exist during a swap operation does not return the variant of `index_not_found` returning the list of non-existent indexes.

**To Reproduce**

Try to exchange several indexes that do not exist.

## Query

`POST`- `swap-indexes`

> `test2` and `test` do not exist.

```json
[
    {
        "swap": ["test2", "test"]
    }
]
```

## Task `error`

The asynchronous `error` is using the variant for a single missing index.

```json
"error": {
                "message": "Index `test2` not found.",
                "code": "index_not_found",
                "type": "invalid_request",
                "link": "https://docs.meilisearch.com/errors#index_not_found"
 },
```

**Expected behavior**

I should receive the message variant listing the indexes that do not exist.

```json
"error": {
                "message": "Indexes `test2`, `test` not found.",
                "code": "index_not_found",
                "type": "invalid_request",
                "link": "https://docs.meilisearch.com/errors#index_not_found"
 },
```

**Meilisearch version:** `v0.30.0-new-index-scheduler.beta.1`

**Comment by loiclec** [comment]
Thanks for the bug report. The problem is that the error happens asynchronously at all. I am on it :)

**Comment by gmourier** [comment]
Thanks @loiclec !

**Comment by loiclec** [comment]
My bad, I have just realised the error should indeed be asynchronous, since we can't know whether the index exists before the index swap is actually processed (there might be an index creation task sent in between the register of the index swap and its processing!). I made a new PR to fix it: https://github.com/meilisearch/meilisearch/compare/index-scheduler...index-swap-error-handling?expand=1

**Comment by curquiza** [comment]
Fixed by #2763

**Comment by Kerollmops** [comment]
Will be fixed accordingly by #2981
